src/lib.rs

```diff
@@ -215,13 +220,24 @@ fn respond_to_request(stream: &mut TcpStream, build_dir_path: impl AsRef<Path>)
 }
 
 #[derive(Debug, StructOpt)]
-pub struct Watch {}
+pub struct Watch {
+    #[structopt(long, short = "w")]
+    watch_paths: Vec<PathBuf>,
+}
 
 impl Watch {
+    pub fn new(watch_paths: Vec<impl AsRef<Path>>) -> Self {
+        Self {
+            watch_paths: watch_paths
+                .iter()
+                .map(|path| path.as_ref().to_owned())
+                .collect(),
+        }
+    }
```

Comment:
I guess it's time we introduce builders... I thought we would at some point. Check [typed-builder](https://crates.io/crates/typed-builder). Try to write one by hand if you are not sure what a builder is.

Reply:
After discussion we noticed it won't be really useful here. Also it adds a dependency which we want to avoid.